README.md

```diff
@@ -580,6 +583,28 @@ enum MyEnum {
 }
 ```
 
+### forbidReturnValueInYieldingMethod
+- Disallows returning values in yielding methods unless marked to return Generator as the value is accessible only via [Generator::getReturn](https://www.php.net/manual/en/generator.getreturn.php)
+- To prevent misuse, this rule can be configured to even stricter mode where it reports such returns regardless of return type declared
```

Comment:
Looking at this for second time, I find this as such a niche use-case that I'd make it strict by default with option to disable it.

With PHPStan error identifiers, it's sounds like a good tradeoff (to whitelist **proper** usages)